src/rust/cryptography-x509-verification/src/policy/extension.rs

```diff
@@ -642,7 +642,7 @@ mod ca {
                 Ok(())
             } else {
                 Err(ValidationError::new(ValidationErrorKind::Other(
-                    "required EKU not found".to_string(),
+                    "EKU and anyEKU not found".to_string(),
```

Comment:
I think this is more accurately an "or"

Reply:
The error case should be reached only if both are missing?

Reply:
Correct, but to me "EKU and anyEKU" can be mis-read as "we didn't find _both_"

Reply:
"nor" instead of "or", but that sounds good to me.

Thank you!

Reply:
Made the changes. It's probably worth squashing all the commits to avoid noise.